Tests/DependencyInjection/HWIOAuthExtensionTest.php

```diff
@@ -343,6 +420,61 @@ public function provideInvalidData()
         );
     }
 
+    public function testCreateResourceOwnerService()
+    {
+        $extension = new HWIOAuthExtension();
+        $extension->createResourceOwnerService($this->containerBuilder, 'my_github', array(
+            'type'              => 'github',
+            'client_id'         => '42',
+            'client_secret'     => 'foo',
+        ));
+
+        $definitions = $this->containerBuilder->getDefinitions();
+
+        $this->assertTrue(isset($definitions['hwi_oauth.resource_owner.my_github']));
+        $this->assertEquals('hwi_oauth.abstract_resource_owner.oauth2', $definitions['hwi_oauth.resource_owner.my_github']->getParent());
+        $this->assertEquals('%hwi_oauth.resource_owner.github.class%', $definitions['hwi_oauth.resource_owner.my_github']->getClass());
+
+        $argument2 = $definitions['hwi_oauth.resource_owner.my_github']->getArgument(2);
+        $this->assertEquals('42', $argument2['client_id']);
+        $this->assertEquals('foo', $argument2['client_secret']);
+        $this->assertEquals('my_github', $definitions['hwi_oauth.resource_owner.my_github']->getArgument(3));
+    }
+
+    public function testCreateResourceOwnerServiceWithService()
+    {
+        $extension = new HWIOAuthExtension();
+        $extension->createResourceOwnerService($this->containerBuilder, 'external_ressource_owner', array(
+            'service'              => 'my.service',
+        ));
+
+        $aliases = $this->containerBuilder->getAliases();
+        $this->assertTrue(isset($aliases['hwi_oauth.resource_owner.external_ressource_owner']));
+        $this->assertEquals('my.service', $aliases['hwi_oauth.resource_owner.external_ressource_owner']->__toString());
```

Comment:
You're right,
`$this->assertEquals('my.service',$aliases['hwi_oauth.resource_owner.external_ressource_owner']);` should do the job